src/labkey/query/Rows.ts

```diff
@@ -61,6 +61,11 @@ export interface QueryRequestOptions extends RequestCallbackOptions {
      * See also: [How To Find schemaName, queryName & viewName](https://www.labkey.org/Documentation/wiki-page.view?name=findNames).
      */
     schemaName: string;
+    /**
+     * Whether qus.getRows should be called to return the update/insert results.
```

Comment:
This comment won't make much sense to someone who isn't familiar with how the server works. Consider removing/rewording the bit about "qus.getRows".